profiles/all.yaml

```diff
@@ -1,5 +1,5 @@
 # This file was auto generated and is intended for debugging an entire build, take caution if editing manually.
-plugins: audio_pipeline,debugview,depthai,gldemo,ground_truth_slam,gtsam_integrator,offline_cam,offline_imu,offload_data,offload_vio.device_rx,offload_vio.device_tx,offload_vio.server_rx,offload_vio.server_tx,passthrough_integrator,pose_lookup,pose_prediction,realsense,record_imu_cam,rk4_integrator,timewarp_gl,zed,fauxpose,display_vk,native_renderer,timewarp_vk,vkdemo,openni,record_rgb_depth
+plugins: audio_pipeline,debugview,depthai,gldemo,ground_truth_slam,gtsam_integrator,offline_cam,offline_imu,offload_data,offload_vio.device_rx,offload_vio.device_tx,offload_vio.server_rx,offload_vio.server_tx,passthrough_integrator,pose_lookup,pose_prediction,realsense,record_imu_cam,rk4_integrator,timewarp_gl,zed,fauxpose,display_vk,native_renderer,timewarp_vk,vkdemo,openni,record_rgb_depth,webcam,hand_tracking,hand_tracking.viewer,hand_tracking_gpu,zed.data_injection
 visualizers: openvins
```

Comment:
For all the profiles, I remember that the "visualizers" field was removed in the hand tracking branch. Is it showing up here because the main branch has this field?

Reply:
Yes. It will be formally removed in another PR